cfme/tests/cloud/test_providers.py

```diff
@@ -94,6 +94,43 @@ def test_discovery_password_mismatch_validation_cloud(appliance):
     view.flash.assert_message('Password/Verify Password do not match')
 
 
+@pytest.mark.tier(3)
+@pytest.mark.uncollectif(lambda appliance: appliance.version >= '5.9',
+                        reason='no more support for cloud provider discovery')
+def test_discovery_error_azure_cloud(appliance):
+    """ Test Azure discovery with feck data
```

Comment:
'fake', typo :)

Reply:
Though this might be my new favorite fake curse word, as it sounds just like another four letter F word.